nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -406,14 +403,24 @@ pub async fn save_v1_asset<T: ConnectionTrait + TransactionTrait>(
                 );
                 txn.execute(query).await?;
             }
-            txn.commit().await?;
         }
     }
+
+    txn.commit().await?;
```

Comment:
This was not moved like this in the merged code, but this looks incorrect anyways.  Recommend closing this PR anyways as the change looks to be already incorporated.  But just calling out this one difference.